models/src/main/scala/coop/rchain/models/ParSet.scala

```diff
@@ -36,13 +36,22 @@ object ParSet {
 
   def apply(
       ps: Seq[Par],
-      connectiveUsed: Boolean = false,
-      remainder: Option[Var] = None
+      connectiveUsed: Boolean,
+      remainder: Option[Var]
```

Comment:
May I suggest making `remainder` the 2nd argument on the occasion? :) Up to you, since it's an additional refactoring.

Reply:
It is 2nd argument.